src/utils/controls.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
There's a tracked-controls utils where all the controllers share code already

Reply:
It makes sense for me to separate `tracked-controls` from the higher level controls utils